Loops/Else with loops/tests.py

```diff
@@ -0,0 +1,23 @@
+from test_helper import run_common_tests, passed, failed, get_answer_placeholders
+
+
+def test_window():
```

Comment:
The condition may vary (e.g. at the moment "if i== 3:" won't pass), we need to think of this test and maybe check the output instead of placeholder